external/js/cothority/package.json

```diff
@@ -12,6 +12,7 @@
   "types": "index.d.ts",
   "scripts": {
     "build": "npm run protobuf && tsc",
+    "watch": "tsc --watch",
```

Comment:
FYI, you can use `npm run build -- --watch` to obtain the same expected behavior.

Reply:
Ok, good to know !
I think it doesn't hurt having an explicit command, it can be of a great help for non-initiated javascript makers (like me ^^).

Reply:
I agree, the small the barrier to entry, the more people can participate :)